core/config.go

```diff
@@ -418,6 +421,52 @@ func (c *SufficientChainConfig) WriteToJSONFile(path string) error {
 	return nil
 }
 
+func resolvePath(path, parent string) string {
```

Comment:
I have been bitten by a gotcha or two for functions like this -- would you please include a test?

Reply:
Test added.